src/containerapp/azext_containerapp/containerapp_env_decorator.py

```diff
@@ -69,3 +75,32 @@ def get_argument_enable_workload_profiles(self):
 
     def get_argument_enable_dedicated_gpu(self):
         return self.get_param("enable_dedicated_gpu")
+
+    def get_argument_logs_dynamic_json_columns(self):
+        return self.get_param("logs_dynamic_json_columns")
+
+
+class ContainerappEnvPreviewUpdateDecorator(ContainerAppEnvUpdateDecorator):
+    def construct_payload(self):
+        super().construct_payload()
+
+        self.set_up_app_log_configuration()
+
+    def set_up_app_log_configuration(self):
+        logs_destination = self.get_argument_logs_destination()
+
+        if logs_destination:
+            logs_destination = None if logs_destination == "none" else logs_destination
+            safe_set(self.managed_env_def, "properties", "appLogsConfiguration", "destination", value=logs_destination)
+
+        if self.get_argument_logs_customer_id() and self.get_argument_logs_key():
```

Comment:
Can customer only update `customer_id `or `logs_key`?

Reply:
they have to update them together as what we do in previous code